components/Controllers/AuctionPageContent/AuctionGraph.tsx

```diff
@@ -201,10 +201,10 @@ export function AuctionGraph({
         : 'Buy now: ......';
     else {
       return auctionUnderlyingPrice
-        ? `SOLD: $${formatBigNum(
+        ? `SOLD: ${formatBigNum(
             auctionUnderlyingPrice,
             controller.underlying.decimals,
-          )}`
+          )} WETH`
```

Comment:
We should just start using ETH everywhere, @jringenberg ? But can be a separate PR 
```suggestion
          )} ETH`
```